policyengine_us/parameters/gov/usda/school_meals/categorical_eligibility.yaml

```diff
@@ -5,9 +5,17 @@ values:
     - snap # Supplemental Nutrition Assistance Program.
     - tanf # Temporary Assistance to Needy Families. Currently only available in CA and IL.
     - fdpir # Food Distribution Program on Indian Reservations.
+    - was_in_foster_care
+    - is_homeless
+    - is_runaway_child
+    - is_migratory_child
+    # Also Head Start child
```

Comment:
file an issue and amend the pr title if this pr isn't adding this criterion

Reply:
could you file an issue specifically for head start providing categorical eligibility for free school meals? we can do that independently of the actual head start logic, for example by including a placeholder variable for head start